### PR TITLE
Gh 48 only ring welcome notification on login

### PR DESCRIPTION
**Pull request by lcombs15**
Resolves GH-48

This only shows after successful login now:
image

### CODE REVIEW

File: app/src/main/kotlin/io/imhungry/home/ui/HomeActivity.kt [hunk 1]
```diff
@@ -45,13 +40,6 @@ class HomeActivity : BaseActivity() {
...
     override fun onStart() {
         super.onStart()
-        notificationHelper.sendNotificationNow(
-            "Welcome!",
-            "We hope you're hungry!",
-            null,
-            NotificationPriority.DEFAULT,
-            0
-        )
     }
```
Comment (TripleFrequency) [inline app/src/main/kotlin/io/imhungry/home/ui/HomeActivity.kt]: Delete the onStart function now that it does nothing.

**Comment by lcombs15** [comment]
@tiemonl Would you locally merge this change with your #28 and make sure everything is all good?

**Comment by TripleFrequency** [comment]
@lcombs15 Just merge yours, we'll look at any conflicts with #28 after.

**Comment by lcombs15** [comment]
@TripleFrequency His changes in #28 are how #48 got started. I'm not worried about code conflicts, I'm worried about functionality.

**Comment by TripleFrequency** [comment]
It won't be a problem. The functionality of the nav bar just launches various activities. Given you had the notification in onStart, we were making several instances of the HomeActivity, while before we only had the one.

**Comment by lcombs15** [comment]
Okay, that fine with me.

**Pull request merged**